libs/utils/android/workloads/gmaps.py

```diff
@@ -90,20 +91,33 @@ def run(self, out_dir, location_search, swipe_count=5, collect=''):
         # Allow the activity to start
         sleep(1)
 
-        self.tracingStart()
+        script = TargetScript(self._te)
+        self._log.debug('Accumulating commands')
+
+        for i in range(swipe_count):
+            System.hswipe(script, 20, 80, 100, True)
+            script.commands.append('sleep 1')
```

Comment:
Should not be more safe to keep `commands` private and make client code slightly slimmer by exposing an append method, e.g. I would prefer to use:

```python
script.append('sleep 1')
```